modules/ROOT/pages/apikit-graphql-module-reference.adoc

```diff
@@ -0,0 +1,246 @@
+= APIkit for GraphQL Module Reference
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+include::reuse::partial$beta-banner.adoc[tag="apikit"]
+
+== Configurations
+---
+
+=== Config
+
+Default configuration
+
+==== Parameters
+
+|===
+| Name | Type | Description | Default Value | Required
+| Name | String | The name for this configuration. Connectors reference the configuration with this name. | | *x*
+| API definition | String | Defines the path to the API definition resource, e.g.: resource::7ddfc767-c5b3-4583-9997-a468e251acd2:posts:1.0.0:graphql:zip:api.graphql |  | *x*
+| Introspection Enabled | Boolean | Enables schema introspection. | false | 
+| Max Query Depth Allowed | Number | The maximum depth a query can have. The value must be greater than 1. | 100 | 
+| Max Query Complexity Allowed | Number | The maximum number of data fields a query can include. The value must be greater than 1. | 100 | 
+| Name | String | The identifier of this element is used to reference it in other components. |  | *x*
+|===
+
+==== Associated Operations
+
+* xref:apikit-graphql-module-reference.adoc#route[Route]
+* xref:apikit-graphql-module-reference.adoc#serialize[Serialize]
+* xref:apikit-graphql-module-reference.adoc#serializebatch[Serialize Batch] 
+
+==== Associated Sources
+
+* xref:apikit-graphql-module-reference.adoc#data-fetcher[Data Fetcher] 
+* xref:apikit-graphql-module-reference.adoc#data-loader[Data Loader] 
+
+== Operations
+
+=== Route
+
+`<graphql-router:route>`
+
+==== Parameters
+
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | *x*
+| Content | Any | Content of the message (usually, the payload). | #[payload] | 
+| Output MIME Type | String | The MIME type of the payload that this operation outputs. |  | 
+| Output Encoding | String | The encoding of the payload that this operation outputs. |  | 
+| Config Ref | ConfigurationProvider | The name of the configuration you use to execute this component. |  | *x*
+| Streaming Strategy | 
+* xref:apikit-graphql-module-reference.adoc#repeatable-in-memory-stream[Repeatable In Memory Stream] 
+* xref:apikit-graphql-module-reference.adoc#repeatable-file-store-stream[Repeatable File Store Stream]
+* Non-Repeatable Stream | Configure if repeatable streams should be used and their behavior. |  | 
+| Target Variable | String | The name of a variable on which the operation output is placed. |  | 
+| Target Value | String | An expression that is evaluated against the operation output and the outcome of that expression is stored in the target variable. | #[payload] | 
+|===
+
+==== Output
+
+|===
+| *Type* | Any
+|===
+
+==== For Configurations
```

Comment:
I assume this heading is part of a template.

Reply:
Let me go back to dev team to ask for this. Thanks!

Reply:
It is, but the "for" is not appropriate, so I'll delete it.